website/source/docs/drivers/docker.html.md

```diff
@@ -324,6 +324,32 @@ The `docker` driver supports the following configuration in the job spec.  Only
     }
     ```
 
+* `cap_add` - (Optional) A list of string flags to pass directly to
```

Comment:
> A list of Linux capabilities as strings to pass directly to <link>. Only whitelisted capabilities may be added. The whitelist can be customized using the [`docker.caps.whitelist`](https://github.com/filipochnik/nomad/blob/8f23c958e42b2efa7a2893c3f3ca500b308feab6/website/source/docs/drivers/docker.html.md#client-configuration) key in the client node's configuration.

And then make sure to document [`docker.caps.whitelist`](https://github.com/filipochnik/nomad/blob/8f23c958e42b2efa7a2893c3f3ca500b308feab6/website/source/docs/drivers/docker.html.md#client-configuration) too.

Note you don't need to use the exact text I wrote. Just make sure to mention the whitelist and the word capabilities just in case someone is searching for that particular word. :)